### PR TITLE
Document Array#shift?

### PR DESCRIPTION
**Pull request by porras**

### CODE REVIEW

**Comment by sdogruyol** [comment]
Welcome back @porras :+1:

File: src/array.cr [hunk 1]
```diff
@@ -1507,6 +1507,18 @@ class Array(T)
     ary
   end
 
+  # Removes the first value of `self`, at index 0. This method returns the removed value.
+  # Returns `nil` without raising any error if array is of 0 size.
```
Comment (RX14) [inline src/array.cr]: I'd turn this sentence around, "If the array is empty..."

Comment (porras) [reply src/array.cr]: It would make sense, I just copied it from the documentation of the question-mark-less method. Should I change it there too?

Comment (RX14) [reply src/array.cr]: Yes please.

Comment (porras) [reply src/array.cr]: Thanks for the feedback, I just pushed the change.

**Comment by porras** [comment]
I don't think the build failure is related to the change...

**Pull request merged**